Source/santametricservice/Writers/SNTMetricHTTPWriterTest.m

```diff
@@ -28,15 +28,15 @@ - (void)setUp {
   self.httpWriter = [[SNTMetricHTTPWriter alloc] init];
   self.mockResponses = [[NSMutableArray alloc] init];
 
```

Comment:
Can you add a comment here with context for why this is necessary?

Reply:
This is the best I could come up with, does it make sense?